indexprovider/wrapper.go

```diff
@@ -528,18 +532,22 @@ func (w *Wrapper) IndexerAnnounceLatestHttp(ctx context.Context, announceUrls []
 }
 
 func (w *Wrapper) MultihashLister(ctx context.Context, prov peer.ID, contextID []byte) (provider.MultihashIterator, error) {
-	provideF := func(proposalCid cid.Cid, pieceCid cid.Cid) (provider.MultihashIterator, error) {
+	provideF := func(identifier string, isDD bool, pieceCid cid.Cid) (provider.MultihashIterator, error) {
+		idName := "propCid"
```

Comment:
I think we can simplify by creating a new log variable, eg `idlog := log.With(idName, identifier)`